CMakeLists.txt

```diff
@@ -77,10 +77,17 @@ add_library(${PROJECT_NAME} INTERFACE)
 #  INTERFACE_COMPILE_FEATURES cxx_std_11
 #)
 
+option(USE_SYSTEM_WISE_TL_OPTIONAL "Use a system wise installation of tl-optional" OFF)
```

Comment:
Is it 'system wide' or 'system wise' ? Not sure myself.

Reply:
I think It should be systemwide. 
https://en.wiktionary.org/wiki/systemwide

I  fix the typo